javase/src/main/java/com/google/zxing/client/j2se/ImageReader.java

```diff
@@ -47,7 +50,13 @@ public static BufferedImage readImage(URI uri) throws IOException {
       throw new IOException("Resource not found: " + uri, iae);
     }
     if (result == null) {
-      throw new IOException("Could not load " + uri);
+      Opener opener_ = new Opener();
```

Comment:
I don't mind efforts to expand TIFF support. How about using JAI though? I'm a little less inclined to add a whole new dependency for it, but if we must, a "standard" Java lib might be preferably especially if it integrations with `ImageIO`.